site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -165,12 +165,37 @@ We need a firewall to allow UDP traffic to nodes tagged as `game-server` via por
 next section.
 
 ```bash
-gcloud compute firewall-rules create game-server-firewall \
+gcloud compute firewall-rules create gke-agones-game-server-firewall \
   --allow udp:7000-8000 \
   --target-tags game-server \
   --description "Firewall to allow game server udp traffic"
 ```
 
+### Create an Autopilot mode cluster for Agones
+
+You can read more about choosing a [cluster mode above](#choosing-a-gke-cluster-mode).
```

Comment:
This line, however, belongs in the top-level "Creating the cluster" section, say after old line 179.